llvm/test/CodeGen/SPIRV/literal-struct.ll

```diff
@@ -13,21 +13,23 @@
 
 ; RUN: llc -O0 %s -o - | FileCheck %s
 
-; CHECK: %[[Int:[0-9]+]] = OpTypeInt 32 0
-; CHECK: %[[Int8:[0-9]+]] = OpTypeInt 8 0
-; CHECK: %[[Int8Ptr:[0-9]+]] = OpTypePointer Generic %[[Int8]]
-; CHECK: %[[StructType:[0-9]+]] = OpTypeStruct %[[Int]] %[[Int]] %[[Int8Ptr]]
+; CHECK-DAG: %[[Int:[0-9]+]] = OpTypeInt 32 0
```

Comment:
I'm not sure we need `DAG` suffix for all of these lines, the latest version of the topol. sorting is quite stable in terms of order, so for the dependent types the order should be fixed enough

Reply:
Ok, I'll rearrange the checks.